helm/cluster-azure/templates/_cilium_helmrelease_config.yaml

```diff
@@ -0,0 +1,25 @@
+{{/* Azure-specific cilium Helm values*/}}
+{{/* https://github.com/giantswarm/cilium-app/blob/main/helm/cilium/values.yaml*/}}
+{{- define "azureCiliumHelmValues" }}
```

Comment:
how is this used later on to install `cilium`?

Reply:
it is installed via cluster chart, cluster chart contains all the apps which are installed for all provideers like cilium. the only thing which is different is the configuration of all apps. that's why we add the config and in value schema we enable apps and plus adding the name of to the configuration.

cluster chart will read the configuration, e.g. azureCiliumHelmValues and render the values in the app itself.

Reply:
ok, thank you!